scalikejdbc-syntax-support-macro/src/test/scala/foo/AutoSpec.scala

```diff
@@ -14,9 +14,10 @@ class AutoSpec extends FlatSpec with Matchers with DBSettings {
     def apply(r: ResultName[Organization])(rs: WrappedResultSet): Organization = autoConstruct(rs, r)
   }
 
-  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, gorupId: Long = 0)
+  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, groupId: Long = 0)
   object Person extends SQLSyntaxSupport[Person] {
     def apply(s: SyntaxProvider[Person])(rs: WrappedResultSet): Person = autoConstruct(rs, s, "organization")
+    override lazy val columns = useField[Person]("organization")
```

Comment:
I'd like to seek more preferred name.

- fieldsToColumns
- entityFieldsToColumns
- constructorParamsToColumns
- autoColumns

autoColumns is a little bit misleading one, but it may
be simple and easy to understand.